### PR TITLE
Applied some of FALLTHROUGH macro usage from #30122

### PR DESCRIPTION
**Pull request by hbina**
I am not sure how to deal with third party libraries as the macro is not defined there....and afaik thirdparty libraries usages should be 1 way only.

Addresses #30122.

### CODE REVIEW

**Comment by bojidar-bg** [comment]
I think you should remove the `// fallthrough` comments.

**Review by Anutrix — CHANGES_REQUESTED** [review]
I think you missed one in editor/plugins/sprite_frames_editor_plugin.cpp

File: editor/plugins/tile_map_editor_plugin.cpp [hunk 1]
```diff
@@ -61,7 +61,7 @@ void TileMapEditor::_notification(int p_what) {
 			if (is_visible_in_tree()) {
 				_update_palette();
 			}
-
+			FALLTHROUGH;
 		} // fallthrough
```
Comment (akien-mga) [inline editor/plugins/tile_map_editor_plugin.cpp]: You can remove the comment.

Comment (hbina) [reply editor/plugins/tile_map_editor_plugin.cpp]: Fixed

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**